src/node/hooks/express/apicalls.js

```diff
@@ -18,7 +18,7 @@ var apiCaller = function(req, res, fields) {
     apiLogger.info("RESPONSE, " + req.params.func + ", " + response);
 
     //is this a jsonp call, if yes, add the function call
-    if(req.query.jsonp)
+    if(req.query.jsonp && isVarName(response))
```

Comment:
Shouldn't this be rquire'd, too? :D (👋 )

Reply:
Indeed :) My local testing apparently wasn't good enough and our automated tests didn't catch it.